packages/i18n-calypso/package.json

```diff
@@ -30,6 +30,7 @@
 		"use-subscription": "1.6.0"
 	},
 	"devDependencies": {
+		"@automattic/calypso-build": "workspace:^",
```

Comment:
why do we need to add these in the other packages?

Reply:
iirc we added a bunch of dependencies because esbuild was complaining not being able to resolve those /cc @yuliyan

Reply:
@noahtallen, I had to rebuild the packages and yarn was unable to execute the `transpile` command. Adding the `@automattic/calypso-build` to those packages resolved the problem.

However, this should probably be in a separate PR.